providers/go-feature-flag/pom.xml

```diff
@@ -43,25 +43,13 @@
             <version>2.19.0</version>
         </dependency>
 
-        <dependency>
```

Comment:
You _may_ be missing some jackson deps. I see you using `jackson.databind` and `jackson.annoations`. I suspect that things are working somewhat "by luck" because these are coming in transitively. You might want to check for sure which you use and make sure to add them, and perhaps commonize your version into a shared prop as well (renovate will then bulk update these).